.github/workflows/test.yml

```diff
@@ -0,0 +1,18 @@
+name: Validate JSON Schema files
+on: [push]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install jsonschema
+        run: pip install jsonschema==3.2.0
+      - run: |
+          for i in *.json; do
```

Comment:
I think this might work with the new repo structure if this line is changed to

```
for i in **/*.json; do
```

Reply:
done in [2a30b2c](https://github.com/MobilityData/gbfs-json-schema/pull/54/commits/2a30b2c27d8ed8b759ad8cf65687157a6d7c3dec)